common/app/views/support/ImageProfile.scala

```diff
@@ -172,6 +173,24 @@ object TwitterImage extends OverlayBase64 {
         }
         new ShareImage(image, TwitterShareImageLogoOverlay.isSwitchedOn)
     }
+    def getContentAgeFileName(prefix: String, publicationYear: Int): String = {
+      // WARNING: we have only produced these content age images up to the year 2025
```

Comment:
Greatest caveat here, and probably the dumbest. We have only created images covering publication dates from 1999 to 2025. If you're reading this message in 2026, you will need to create more, or find a programmatic way of applying the overlay.

Reply:
This should be handled by using an SVG!

Reply:
I can't see anything in the [Image Overlay Fastly docs](https://developer.fastly.com/reference/io/overlay/) to suggest it doesn't support SVGs 🙏